community/jmx/src/main/java/org/neo4j/jmx/Kernel.java

```diff
@@ -27,7 +27,7 @@
 @Description( "Information about the Neo4j kernel" )
 public interface Kernel
 {
-    final String NAME = "Kernel";
+    String NAME = "Kernel";
```

Comment:
The _final_ keyword is redundant in an interface. So this is likely IntelliJ's suggested quickfix applied :-)